### DISCUSSION TITLE
Projects are disabled for all repositories in organization

### DISCUSSION THREAD

**Discussion by Th3S4mur41** [community/community#31462] [Projects and Issues]
I have an organization containing a project that I would like to add as a tab to one of the repository.
Although projects are properly activated on organizational level, since I have indeed a working project to prove it. Settings of the repo tell me otherwise:
![image](https://user-images.githubusercontent.com/5543595/187638055-e773167e-3902-4938-b6ce-65730be5bdca.png)

When I click on the settings link, I get to the organization's projects settings which as expected shows me that everything is properly activated

![image](https://user-images.githubusercontent.com/5543595/187638471-d5d979f4-ffc7-4a52-a64c-f0e264fff1a4.png)

**Comment by jfullstackdev** [comment]
make sure you click Save, even if you check it, that will not be updated without clicking Save but I'm not very sure whether it's the situation

I also noticed that the blanks were not of the same length

**Reply by Th3S4mur41** [reply]
I already tried to save... Also disabled saved, then enabled saved again. Refresh, Clear cache, nothing helped.

The difference in the length of the blank is because the first screen uses the display name of the organization while the second one uses the full name

**Reply by danielbroll** [reply]
We have the exact same issue.

**Comment by Th3S4mur41** [comment]
Not sure if a fix was published... But the setting is now available in the repository and the project tab is visible without any change to the configuration 😊
